### PR TITLE
[stable/mission-control] Bumped version and added RBAC support

### PR DESCRIPTION
**Pull request by jainishshah17**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by jainishshah17** [comment]
@lachie83 Can you please take a look?

**Comment by jainishshah17** [comment]
@eldada

**Comment by jainishshah17** [comment]
@viglesiasce Can you please take a look?

File: stable/mission-control/Chart.yaml [hunk 1]
```diff
@@ -10,9 +10,11 @@ maintainers:
...
   name: eldada
+- email: rimasm@jfrog.com
+  name: rimusz
 name: mission-control
 sources:
 - https://bintray.com/jfrog/product/mission-control/view
 - https://github.com/JFrogDev
-version: 0.3.0
-appVersion: 3.0.0
+version: 0.4.1
```
Comment (mattfarina) [inline stable/mission-control/Chart.yaml:19]: Out of curiosity, why is this `0.4.1` and not `0.4.0`?

**Pull request merged**